angular-spa/src/app/app.module.ts

```diff
@@ -0,0 +1,99 @@
+// Required for Angular multi-browser support
+import { BrowserModule } from '@angular/platform-browser';
+
+// Required for Angular
+import { NgModule } from '@angular/core';
+
+// Required modules and components for this application
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { ProfileComponent } from './profile/profile.component';
+import { HomeComponent } from './home/home.component';
+
+// HTTP modules required by MSAL
+import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
+
+// Required for MSAL
+import { IPublicClientApplication, PublicClientApplication, InteractionType, BrowserCacheLocation } from '@azure/msal-browser';
+import { MsalGuard, MsalInterceptor, MsalBroadcastService, MsalInterceptorConfiguration, MsalModule, MsalService, MSAL_GUARD_CONFIG, MSAL_INSTANCE, MSAL_INTERCEPTOR_CONFIG, MsalGuardConfiguration, MsalRedirectComponent } from '@azure/msal-angular';
+
+const isIE = window.navigator.userAgent.indexOf('MSIE ') > -1 || window.navigator.userAgent.indexOf('Trident/') > -1;
+
+export function MSALInstanceFactory(): IPublicClientApplication {
+  return new PublicClientApplication({
+    auth: {
+      // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+      clientId: '',
+    
+      // Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+      authority: '',
+
+      // Must be the same redirectUri as what was provided in your AD app registration.
+      redirectUri: '/'

```

Comment:
Here you have `'/'` and in the app registration you have `http://local...` -- if we want them to put the http://localhost value in here, then we probably shouldn't have `'/'` here.  If instead we want `'/'` here, then the comment shouldn't suggest that users overwrite it.  Can you clarify what you'd like the user to do here.

Reply:
Thanks! Addressed.

Reply:
Thanks!